persistence/sql/persister_session.go

```diff
@@ -36,7 +36,8 @@ func (p *Persister) GetSession(ctx context.Context, sid uuid.UUID, expandables s
 	nid := p.NetworkID(ctx)
 
 	q := p.GetConnection(ctx).Q()
-	if len(expandables) > 0 {
+	// if len(expandables) > 0 {
+	if expandables.Has(session.ExpandSessionDevices) {
```

Comment:
This is a temporary hack to get around the `Devices` eager query despite `expandables.ToEager()...` returning an empty slice (`Identity` expandable does not count as a regular expandable).

Reply:
@zepatrik context to the slack conversation ^